Jenkinsfile

```diff
@@ -62,7 +62,7 @@ pipeline {
             emailext (
                 body: "${currentBuild.currentResult}: Job ${env.JOB_NAME} build ${env.BUILD_NUMBER}\n More info at: ${env.BUILD_URL}",
                 subject: "Jenkins Build ${currentBuild.currentResult}: Job ${env.JOB_NAME}",
-                to: 'micael.oliveira@anu.edu.au'
+                to: 'micael.oliveira@anu.edu.au, aidan.heerdegen@anu.edu.au'
```

Comment:
Did you mean for this to be Andrew rather than me?

Reply:
Oh, sorry, I thought you were the one who replied to the comment!

Reply:
I'll change it.